src/ncursesw.h

```diff
@@ -405,6 +405,12 @@ inline int UNDEF( insertln )()	{ return insertln(); }
 #define insertln UNDEF(insertln)
 #endif
 
+#ifdef is_linetouched
```

Comment:
Note, I took this code from upstream ncursesw. To be honest I do not know why we have partial copy of that file there we just have subset of this macro redefinition and modify some method signatures to act as shortcut, but overall I do not believe it is proper way how to do it.